### ISSUE TITLE
Add Dino's Offline Adventure to Open Source ROMS

### ISSUE THREAD

**Issue by torch2424** [torch2424/wasmBoy#195]
Should also do this in vaporboy.

https://gaming-monster.itch.io/dinos-offline-adventure

Got permission:

<img width="711" alt="screen shot 2018-11-12 at 1 41 36 am" src="https://user-images.githubusercontent.com/1448289/48339138-466fb980-e61c-11e8-901c-ccc12049fcd5.png">
<img width="703" alt="screen shot 2018-11-12 at 1 41 45 am" src="https://user-images.githubusercontent.com/1448289/48339139-466fb980-e61c-11e8-8f9b-c552da4afc70.png">
<img width="712" alt="screen shot 2018-11-12 at 1 42 09 am" src="https://user-images.githubusercontent.com/1448289/48339140-47085000-e61c-11e8-821f-9da6987bbf77.png">

**Comment by gingemonster** [comment]
Yup just to confirm Im happy for it to be included, obviously a credit would be nice

**Comment by torch2424** [comment]
100% planned on giving credit 😄 I try to do that on everything I use in my personal projects.

The github syntax highlighing isn't that great, but here's where I store all of my public ROMs: https://github.com/torch2424/wasmBoy/blob/master/demo/debugger/wasmboyFilePicker/openSourceROMs.js

Where each has an `infoElement`, giving credit and background on the ROM itself 😄
